Uni_life_tracker/app/src/main/res/layout/activity_expenses_landing.xml

```diff
@@ -0,0 +1,132 @@
+
+        <com.google.android.material.textfield.TextInputEditText
+                android:layout_width="240dp"
+                android:layout_marginTop="250dp"
+                android:layout_marginStart="95dp"
+                android:layout_height="wrap_content"
+                android:hint="@string/add_name"
+                android:inputType="text"
+        />
+        <com.google.android.material.textfield.TextInputLayout
+                android:id="@+id/e_add_amount"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintRight_toRightOf="parent"
+                app:errorEnabled="true">
+        </com.google.android.material.textfield.TextInputLayout>
+
+
+        <com.google.android.material.textfield.TextInputEditText
+                android:layout_width="240dp"
+                android:layout_marginStart="95dp"
+                android:layout_height="wrap_content"
+                android:hint="@string/add_amount"
+                android:inputType="text"
+        />
+        <com.google.android.material.textfield.TextInputLayout
+                android:id="@+id/e_add_date"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintRight_toRightOf="parent"
+                app:errorEnabled="true">
+        </com.google.android.material.textfield.TextInputLayout>
+
+
+        <com.google.android.material.textfield.TextInputEditText
+                android:layout_width="240dp"
+                android:layout_marginStart="95dp"
+                android:layout_height="wrap_content"
+                android:hint="@string/add_date"
+                android:inputType="text"
+        />
+        <ImageButton
+                android:id="@+id/plus_button"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintRight_toRightOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                android:layout_width="59dp"
+                android:layout_height="65dp"
+                android:src="@drawable/plus_button_round"
+                android:padding="12dp"
+                android:scaleType="fitCenter"
+                android:background="#0000ffff"
+                app:layout_constraintHorizontal_bias="0.049"
+                app:layout_constraintVertical_bias="0.0"
+                app:layout_constraintStart_toStartOf="parent"
+                android:layout_marginStart="180dp"
+                android:contentDescription="@string/left_arrow"/>
```

Comment:
If I'm not mistaken, I don't think the plus button is doing anything right now. I was wondering if you could code the whole duplication thing so that you could add more than one expense

Reply:
I think it might be simpler to remove the plus button for now and then after clicking the button, take the user to expenses list so they can then choose to add another expense from there.